mobidam-sst-management-frontend/frontend/src/components/AddPersonDialog.vue

```diff
@@ -0,0 +1,193 @@
+                                    color="primary"
+                                    header-color="primary"
+                                    :first-day-of-week="1"
+                                    :min="new Date().toISOString()"
+                                ></v-date-picker>
+                            </v-menu>
+                        </v-col>
+                    </v-row>
+                </v-form>
+            </v-card-text>
+            <v-divider class="divider"></v-divider>
+            <v-card-actions>
+                <v-btn
+                    text
+                    @click="closeDialog"
+                    >Schließen</v-btn
+                >
+                <v-spacer></v-spacer>
+                <v-btn
+                    class="white--text"
+                    color="success"
+                    @click="saveTask"
+                >
+                    Speichern
+                </v-btn>
+            </v-card-actions>
+        </v-card>
+    </v-dialog>
+</template>
+
+<script setup lang="ts">
+import Zuordnung from "@/types/Zuordnung";
+import { ref, reactive } from "vue";
+import ZuordnungService from "@/api/ZuordnungService";
+import { useRouter } from "vue-router/composables";
+import { useRules } from "@/composables/rules";
+
+const maxLength = 255;
+const validationRules = useRules();
+const textInputRules = [
+    validationRules.notEmptyRule("Das Feld darf nicht leer sein."),
+    validationRules.maxLengthRule(
+        maxLength,
+        "Die Eingabe darf maximal " + maxLength + " Zeichen lang sein."
+    ),
+];
+
+interface Props {
+    showDialog: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+    showDialog: false,
+});
+
+const newZuordnung: Zuordnung = reactive({
+    schnittstelle: "",
+    userID: "",
+    department: "",
+    functionAddress: "",
+    validFrom: "",
+    validUntil: "",
+});
+
+const emit = defineEmits<{
+    (e: "update:showDialog", b: boolean): void;
+    (e: "zuordnung-saved"): void;
+}>();
+
+const form = ref<HTMLFormElement>();
+let schnittstelleID = useRouter().currentRoute.params.id;
+
+function saveTask(): void {
+    if (form.value?.validate()) {
+        newZuordnung.schnittstelle = schnittstelleID;
+        ZuordnungService.create(newZuordnung)
+            .then(() => {
+                closeDialog();
+                resetZuordnung();
+                emit("zuordnung-saved");
+            })
+            .finally(() => {
+                form.value?.resetValidation();
+            });
+    }
+}
+
+function resetZuordnung(): void {
+    newZuordnung.schnittstelle = "";
+    newZuordnung.userID = "";
+    newZuordnung.department = "";
+    newZuordnung.functionAddress = "";
+    newZuordnung.validFrom = "";
+    newZuordnung.validUntil = "";
+}
+
+function closeDialog() {
+    emit("update:showDialog", false);
+    form.value?.resetValidation();
+}
+</script>
+
+<style scoped>
+.divider {
+    background-color: #395b64;
```

Comment:
Farben würde ich immer in ein CSS File auslagern sonst wird Design-Überarbeitung ziemlich schwer